src/clojure/nrepl/cmdline.clj

```diff
@@ -269,7 +269,7 @@
             greeting-fn (if (= transport #'transport/tty) #'transport/tty-greeting)
             server (start-server :port port :bind bind :handler handler
                                  :transport-fn transport :greeting-fn greeting-fn)]
-        (when-let [ack-port (:ack options)]
+        (when-let [ack-port (some-> (:ack options) ->int)]
```

Comment:
I guess this function is safe to call on a real integer, right?

Reply:
Yes, it checks if it's a number before parsing, it's used for `:port` param

Reply:
👍